### DISCUSSION TITLE
Unable to run jupyter notebook cell

### DISCUSSION THREAD

**Discussion by NevilleMthw** [microsoft/vscode-jupyter#10972] [Questions and Answers] [answered]
Hi, I am facing an issue trying to run a jupyter notebook cell in vscode, I am just trying to use a simple Python script along with CSV data and images.

This is the error I am getting for when I run my cell:

![image](https://user-images.githubusercontent.com/18580821/181659686-2d11256c-8a0f-49f0-9308-36e1def9b88a.png)

The data I am dealing with is 6000+ images with the same for CSV rows. 
Also, to note I am running this script on remote ssh to my university machine. However, when I run it on my local machine, it shows no error. I guess the issue is with the size of my data but I just want to know if there is a workaround because the machine I am connected to is quite powerful and has good memory as well, so I don't see why I should be getting this issue.

The Python script:

```
for index, row in train_labels_csv.iterrows():
    
    (filename, width, height, class_name, xmin, ymin, xmax, ymax) = row
    
    train_image_fullpath = os.path.join(train_path, filename)
    train_img = keras.preprocessing.image.load_img(train_image_fullpath, target_size=(height, width))
    train_img_arr = keras.preprocessing.image.img_to_array(train_img)
    
    xmin = xmin / width
    ymin = ymin / height
    xmax = xmax / width
    ymax = ymax / height
    # xmin = round(xmin/ width, 2)
    # ymin = round(ymin/ height, 2)
    # xmax = round(xmax/ width, 2)
    # ymax = round(ymax/ height, 2)
    
    train_images.append(train_img_arr)
    train_targets.append((xmin, ymin, xmax, ymax))
    train_labels.append(class_name)
```

**Comment by rchiodo** [comment, marked as answer]
I'm guessing you're getting an out of memory exception. Unfortunately there's nothing we can do about this. You're best bet is to make sure the python environment on the university machine has all of the latest packages for what you're using.

**Reply by NevilleMthw** [reply]
Thanks for the reply. @rchiodo 
I just changed up the code a bit to not use the Keras library and the out of memory issue did not happen again.
